solutions/usaco-858.mdx

```diff
@@ -134,88 +153,75 @@ public class convention {
 	}
 }
 ```
-
 </JavaSection>
-
 <CPPSection>
 
 ```cpp
+#include <algorithm>
+#include <cstdio>
```

Comment:
is this necessary tho?

Reply:
`algorithm` for sort
`cstdio` for freopen